src/Microsoft.DocAsCode.MarkdigEngine.Extensions/TripleColon/ImageExtension.cs

```diff
@@ -110,7 +109,11 @@ public bool TryProcessAttributes(IDictionary<string, string> attributes, out Htm
 
                 if (!string.IsNullOrEmpty(currentLightbox))
                 {
-                    renderer.WriteLine($"<a href=\"{currentLightbox}#lightbox\" data-linktype=\"relative-path\">");
+                    var lighboxHtmlAttributes = new HtmlAttributes();
+                    lighboxHtmlAttributes.AddProperty("href", $"{currentLightbox}#lightbox");
+                    lighboxHtmlAttributes.AddProperty("data-linktype", $"relative-path");
+                    //renderer.WriteLine($"<a href=\"{currentLightbox}#lightbox\" data-linktype=\"relative-path\">");
```

Comment:
Hi @adunndevster , should this commented code be removed or kept as a reference?  Thanks

Reply:
Oh, lemme remove!